osu.Game.Rulesets.tau/UI/TauPlayfieldAdjustmentContainer.cs

```diff
@@ -30,5 +30,20 @@ public TauPlayfieldAdjustmentContainer()
                 Child = content = new ScalingContainer { RelativeSizeAxes = Axes.Both }
             };
         }
+
+        /// <summary>
+        /// A <see cref="Container"/> which scales its content relative to a target width.
+        /// </summary>
+        private class ScalingContainer : Container
+        {
+            protected override void Update()
+            {
+                base.Update();
+
+                Scale = new Vector2(Parent.ChildSize.X / TauPlayfield.BASE_SIZE.X);
+
+                Size = Vector2.Divide(Vector2.One, Scale);
+            }
+        }
```

Comment:
What's this here for?

Reply:
It makes the size value stay the same to make sure replay inputs always get correctly converted to screen space

Reply:
oh wait nevermind i didn't see that this was overriding the `ScalingContainer` class from osu's main code